pom.xml

```diff
@@ -11,7 +11,7 @@
 	<groupId>io.jenkins.services.distribution-service</groupId>
 	<artifactId>custom-distribution-service</artifactId>
 	<version>1.0.0</version>
```

Comment:
Normally version `1.0.0` is a release one that has the basic functions.

Reply:
Yeah agreed, so should we put it as 0.0.1 ?

Reply:
`0.0.1` is better.

Reply:
Affirmative :+1: